Firestore/Swift/Tests/Codable/FirestoreEncoderTests.swift

```diff
@@ -609,17 +609,20 @@ class FirestoreEncoderTests: XCTestCase {
 private func assertThat(_ dictionary: [String: Any],
                         in document: String? = nil,
                         file: StaticString = #file,
-                        line: UInt = #line) -> DictionarySubject {
+                        line: UInt = #line) -> DictionarySubject
+{
```

Comment:
Was there a configuration change associated with this? Common Swift style (and Google's go/swift-style specify that braces should trail rather than start a new line.

Reply:
Nope, it looks like we might need to add an option in that case.

Reply:
Resolved via https://github.com/nicklockwood/SwiftFormat/issues/696